src/localization/strings.ts

```diff
@@ -21,6 +21,11 @@ export interface Localized extends LocalizedStringsMethods {
     home: string;
     admin: string;
     onCall: string;
+    balance: string;
+    pages: string[];
```

Comment:
true, I looked at them but couldn't figure out how to make the enum work in my code in a timely manner so I opted for this. Then again, there's no nav-items currently so this can go.